server/src/Dockerfile

```diff
@@ -18,29 +18,7 @@ ENV PATH /usr/local/bin:$PATH
 # > At the moment, setting "LANG=C" on a Linux system *fundamentally breaks Python 3*, and that's not OK.
 ENV LANG C.UTF-8
 
-# runtime dependencies
-RUN set -eux \
-	&& apt-get update \
-	&& apt-get install -y --no-install-recommends \
-		ca-certificates \
-		netbase \
-	&& rm -rf /var/lib/apt/lists/*
-
-ENV GPG_KEY 3.8.1
-ENV PYTHON_VERSION 3.8.1
-
-EXPOSE 80
-
-RUN set -ex \
-	 \
-	&& savedAptMark="$(apt-mark showmanual)" \
-	&& apt-get update && apt-get install -y --no-install-recommends \
-		gcc \
-		python3-dev \
-		libffi-dev \
-		libssl-dev \
-	&& pip install --upgrade pip setuptools wheel \
-    && pip install -r /usr/src/app/requirements.txt \
```

Comment:
Is this because pyro-engine image already has all this?

Reply:
Yes :)